Modules/TPC/include/TPC/CheckOfSlices.h

```diff
@@ -44,17 +44,33 @@ class CheckOfSlices : public o2::quality_control::checker::CheckInterface
   std::string getAcceptedType() override { return "TCanvas"; }
 
  private:
-  ClassDefOverride(CheckOfSlices, 2);
+  ClassDefOverride(CheckOfSlices, 3);
+  void calculateStatistics(const double* yValues, const double* yErrors, bool useErrors, const int firstPoint, const int lastPoint, double& mean, double& stddevOfMean);
```

Comment:
I think this is copy'n'paste of `CheckOfTrendings::calculateStatistics`
Please put it in one common helper file e.g. `include/TPC/Utility.h`

Reply:
I moved `calculateStatistics` to` include/TPC/Utility.h` and removed it from both `CheckOfTrendings` and `CheckOfSlices`